tmt/export.py

```diff
@@ -46,21 +46,22 @@ def export_to_nitrate(test, create, general):
     try:
         nitrate_id = test.node.get('extra-nitrate')[3:]
         nitrate_case = nitrate.TestCase(int(nitrate_id))
-        nitrate_case.summary # Make sure we connect to the server now
+        nitrate_case.summary  # Make sure we connect to the server now
         echo(style(f"Test case '{nitrate_case.identifier}' found.", fg='blue'))
     except TypeError:
         # Create a new nitrate test case
         if create:
             nitrate_case = create_nitrate_case(test)
             new_test_created = True
+            test._metadata['extra-summary'] = nitrate_case.summary
```

Comment:
IMO creating memory-only attributes (no backing in fmf files) should be avoided.

Reply:
This is an oversight, it should be written in fmf files. I will fix it.

Reply:
I'm in black fan group which produces 
```
+    summary = (
+        test._metadata.get("extra-summary")
+        or test._metadata.get("extra-task")
+        or test.summary
+        or ""
+    )
```
Call on @psss about his preference.

Reply:
Should be fixed by 16a2608 
It's not ideal solution (another append), however it will hopefully do for the time being.